api/src/opentrons/protocol_api/instrument_context.py

```
@@ -1464,7 +1471,7 @@ def trash_container(self) -> labware.Labware:
         return self._trash
 
     @trash_container.setter
-    def trash_container(self, trash: labware.Labware) -> None:
+    def trash_container(self, trash: Optional[labware.Labware]) -> None:
```

This is an API change also. It's probably fine because it's more permissive rather than less, but it probably should raise an exception if the api version is below that version in which we added the possibility for this to be none. I think it's probably also a fine idea to not make this change, since I don't quite get why someone would want to explicitly remove a trash from a pipette, or have a different API for that.